src/amp/components/Epic.tsx

```diff
@@ -187,117 +187,123 @@ const buildUrl = (
-									<p className={topRightStyle}>
-										{moustacheVariable('topRight')}
-									</p>
-									<p className={labelStyle}>
-										{moustacheVariable('bottomRight')}
-									</p>
-								</div>
-							</div>
 
-							<div>
-								<div className={tickerBackgroundStyle}>
-									<MoustacheSection name="goalExceededMarkerPercentage">
+								<div>
+									<div className={tickerBackgroundStyle}>
+										<MoustacheSection name="goalExceededMarkerPercentage">
+											<div
+												id="goal-exceeded-marker"
+												className={
+													goalExceededMarkerStyle
+												}
+												style={{
+													left: `${moustacheVariable(
+														'goalExceededMarkerPercentage',
+													)}%`,
+												}}
+											/>
+										</MoustacheSection>
+
 										<div
-											id="goal-exceeded-marker"
-											className={goalExceededMarkerStyle}
+											id="ticker-progress"
+											className={tickerProgressStyle}
 											style={{
-												left: `${moustacheVariable(
-													'goalExceededMarkerPercentage',
+												width: `${moustacheVariable(
+													'percentage',
 												)}%`,
 											}}
 										/>
-									</MoustacheSection>
-
-									<div
-										id="ticker-progress"
-										className={tickerProgressStyle}
-										style={{
-											width: `${moustacheVariable(
-												'percentage',
-											)}%`,
-										}}
-									/>
+									</div>
 								</div>
 							</div>
-						</div>
-					</MoustacheSection>
-
-					<h2 className={epicHeaderStyle}>
-						<MoustacheVariable name="heading" />
-					</h2>
-					<MoustacheSection name="paragraphs">
-						<p className={epicParagraphStyle}>
-							<MoustacheVariable name="." />
-						</p>
-					</MoustacheSection>
-					<span className={highlightedTextStyle}>
-						<MoustacheVariable name="highlightedText" />
-					</span>
-					<br />
-					<MoustacheSection name="cta">
-						<a
-							href={buildUrl(
-								moustacheVariable('url'),
-								webURL,
-								moustacheVariable('campaignCode'),
-								moustacheVariable('componentId'),
-							)}
-							className={supportButtonStyle}
-						>
-							<MoustacheVariable name="text" />
-							<svg
-								className={arrowStyle}
-								xmlns="http://www.w3.org/2000/svg"
-								viewBox="0 0 20 17.89"
-								preserveAspectRatio="xMinYMid"
-								aria-hidden="true"
-								focusable="false"
+						</MoustacheSection>
+						<h2 className={epicHeaderStyle}>
+							<MoustacheVariable name="heading" />
+						</h2>
+						<MoustacheSection name="paragraphs">
+							<p className={epicParagraphStyle}>
+								<MoustacheVariable name="." />
+							</p>
+						</MoustacheSection>
+						<span className={highlightedTextStyle}>
+							<MoustacheVariable name="highlightedText" />
+						</span>
+						<br />
+						<MoustacheSection name="cta">
+							<a
+								href={buildUrl(
```

Comment:
the `buildUrl` function now needs to add the `abTest` field, so we can track acquisitions against the test

Reply:
One other thing - is it possible to add the `referrerPageviewId` field in as well? It's not essential, but if possible it would be good to pass through the ophan page view id

Reply:
Sorted ✅